airflow/serialization/serialized_objects.py

```diff
@@ -586,9 +604,24 @@ def serialize_operator(cls, op: BaseOperator) -> Dict[str, Any]:
         return serialize_op
 
     @classmethod
-    def deserialize_operator(cls, encoded_op: Dict[str, Any]) -> BaseOperator:
+    def deserialize_operator(cls, encoded_op: Dict[str, Any]) -> Union[BaseOperator, MappedOperator]:
         """Deserializes an operator from a JSON object."""
-        op = SerializedBaseOperator(task_id=encoded_op['task_id'])
+        op: Union[BaseOperator, MappedOperator]
+        # Check if it's a mapped operator
+        if "mapped_kwargs" in encoded_op:
+            op = MappedOperator(
+                task_id=encoded_op['task_id'],
+                dag=None,
+                operator_class='.'.join(filter(None, (encoded_op['_task_module'], encoded_op['_task_type']))),
+                # These are all re-set later
+                partial_kwargs={},
```

Comment:
Shouldn't `partial_kwargs={}` be a default on the `MappedOperator` itself?

Reply:
No, cos in all other situations where this is constructor we need to pass a value.